Dockerfile

```diff
@@ -1,96 +1,70 @@
 ### Pre-install yum stuff
-ARG BASE_IMAGE=quay.io/app-sre/ubi8-ubi-minimal:8.7-923
+ARG BASE_IMAGE=registry.access.redhat.com/ubi9/ubi-minimal:9.1.0
 FROM ${BASE_IMAGE} as base-update
 
-RUN microdnf --assumeyes update \
+RUN microdnf --assumeyes --nodocs update \
       && microdnf clean all \
       && rm -rf /var/yum/cache
 
 FROM base-update as dnf-install
 
-# Replace version with a version number to pin a specific version (eg: "-123.0.0")
-ARG GCLOUD_VERSION=
-
 # OCM backplane console port to map
 ENV OCM_BACKPLANE_CONSOLE_PORT 9999
 
-# install gcloud-cli
-RUN mkdir -p /gcloud/bin
-ENV CLOUDSDK_PYTHON=/usr/bin/python3.6
-WORKDIR /gcloud
+# Add google repo
 COPY utils/dockerfile_assets/google-cloud-sdk.repo /etc/yum.repos.d/
 
+RUN rpm --import https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-9 \
+      && rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+
 # Install packages
 # These packages will end up in the final image
 # Installed here to save build time
-RUN microdnf --assumeyes install \
-    bash-completion \
-    findutils \
-    git \
-    golang \
-    google-cloud-sdk${GCLOUD_VERSION} \
-    jq \
-    make \
-    openssl \
-    procps-ng \
-    python36 \
-    python39 \
-    python39-pip \
-    rsync \
-    tar \
-    vim-enhanced \
-    wget \
-    && microdnf clean all \
-    && rm -rf /var/yum/cache
-    && update-alternatives --set python3 /usr/bin/python3.9;
-
-RUN curl -sSlo epel-gpg https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8 \
-    && rpm --import epel-gpg \
-    && rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm \
-    && microdnf --assumeyes install \
-         sshuttle \
-         lnav \
-    && microdnf clean all
+RUN microdnf --assumeyes --nodocs install \
+      bash-completion \
+      findutils \
+      git \
+      golang \
+      google-cloud-cli \
+      jq \
+      make \
+      nodejs \
+      nodejs-nodemon \
+      npm \
+      openssl \
+      procps-ng \
+      python3 \
+      python3-pip \
+      rsync \
+      sshuttle \
+      tar \
+      vim-enhanced \
+      wget \
+      && microdnf clean all \
+      && rm -rf /var/yum/cache
```

Comment:
This may need that `\`.
The build succeeded using `--no-cache` without it so it may not be necessary.

Reply:
It won't need the \.  It's the last on the line.

Reply:
That would explain why it built properly. :)